tests/console/yast2_dns_server.pm

```diff
@@ -25,10 +25,10 @@ sub assert_running() {
     my $running = shift;
 
     if ($running) {
-        assert_script_run 'systemctl is-active named | grep -E "^active"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^active"';
     }
     else {
-        assert_script_run 'systemctl is-active named | grep -E "^(inactive|unknown)"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^(inactive|unknown)"';
```

Comment:
You used this with is-enabled check, why not here? (assert_script_run "! systemctl is-active named")